generator/source/faq/docker.rst

```diff
@@ -221,6 +221,15 @@ I get this error running Docker: ``ERROR: manifest for hyperledger/sawtooth-vali
 -----------------------------------------------------------------------------------------------------
 You are following instructions for the unreleased Sawtooth ``nightly`` build. There are no Docker images for the nightly build. Instead use the ``latest`` build documentation at https://sawtooth.hyperledger.org/docs/core/releases/latest/app_developers_guide.html
 
+sawtooth-default-poet.yaml doesnt start the network successfully on subsequent runs ?
```

Comment:
I suggest:
`Why doesn't sawtooth-default-poet.yaml start the network successfully on subsequent runs?`